src/openrct2/ride/Ride.h

```diff
@@ -232,7 +233,7 @@ struct Ride
     uint8_t proposed_num_cars_per_train;
     uint8_t max_trains;
 
-private:
+public: // private:
     uint8_t min_max_cars_per_train;
```

Comment:
This variable needs to be either split so that it can actually properly save the two, or removed altogether.